back/src/Services/SocketManager.ts

```diff
@@ -1500,6 +1589,106 @@ export class SocketManager {
             room.dispatchEvent(name, value, "RoomApi", []);
         }
     }
+
+    handleKickOffUserMessage(user: User, userKickedUuid: string) {
+        const group = user.group;
+        if (!group) {
+            return;
+        }
+        const usersKiked = group.getUsers().filter((user) => user.uuid === userKickedUuid);
+        if (usersKiked.length === 0) return;
+        for (const userKiked of usersKiked) {
+            group.leave(userKiked);
+        }
+        // TODO fixme to notify only user kiked
+        group.setOutOfBounds(true);
+    }
+
+    handerMuteParticipantIdSpaceMessage(user: User, userMutedUuid: string) {
+        const group = user.group;
+        if (!group) {
+            return;
+        }
+        const usersMuted = group.getUsers().filter((user) => user.uuid === userMutedUuid);
+        for (const userMuted of usersMuted) {
+            // send mute event
+            const serverToClientMessage: ServerToClientMessage = {
+                message: {
+                    $case: "mutedMessage",
+                    mutedMessage: {
+                        userUuid: user.uuid,
+                        message: "muted",
+                    },
+                },
+            };
+            userMuted.socket.write(serverToClientMessage);
+        }
+    }
+
+    handlerMuteEveryBodySpaceMessage(user: User) {
+        const group = user.group;
+        if (!group) {
+            return;
+        }
+        for (const userMuted of group.getUsers().values()) {
```

Comment:
That would be `mutedUser` instead of `userMuted`. In English, the adjective goes before the noun.